gr-utils/python/modtool/cli/add.py

```diff
@@ -36,7 +36,7 @@
 
 @click.command('add')
 @click.option('-t', '--block-type', type=click.Choice(ModToolAdd.block_types),
-              help="One of {}.".format(', '.join(ModToolAdd.block_types)))
+              help=f"One of {', '.join(ModToolAdd.block_types)}."
```

Comment:
Here is a `)` missing.

Reply:
Oh! Syntax errors; that indicates `gr_modtool` has not  been actually exercised.